src/main/resources/psw4j.properties

```diff
@@ -0,0 +1,25 @@
+### Argon2
+hash.argon2.memory=1024
+hash.argon2.iterations=20
+hash.argon2.length=128
+hash.argon2.parallelism=4
+hash.argon2.type=id
+global.banner=false
+global.salt.length=14
+
+
+### bcrypt
+hash.bcrypt.minor=b
+# logarithmic cost (cost = 2^12)
+hash.bcrypt.rounds=12
+
+
+### scrypt
```

Comment:
Don't need `scrypt` and `bcrypt` options

Reply:
Fixed, thank you.